spinalcordtoolbox/download.py

```diff
@@ -40,7 +40,9 @@
     },
     "sct_testing_data": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/sct_testing_data/releases/download/r20230207/sct_testing_data-r20230207.zip",
+            # TODO: Replace with stable release generated by distriploy once PR is merged:
+            #       https://github.com/spinalcordtoolbox/sct_testing_data/pull/12
+            "https://github.com/spinalcordtoolbox/sct_testing_data/archive/jn/update-for-sct-4302-4345.zip",
```

Comment:
Side note: Please disregard the test failures! This is because the `sct_testing_data` changes also include changes meant for #4345.

Reply:
Resolved by merging master and updating the sct_testing_data link!